### PR TITLE
change: calendar UI and version number

### PR DESCRIPTION
**Pull request by khushbuSatasiya**
change: calendar UI

### CODE REVIEW

File: src/features/checkIn/component/checkIn.tsx [hunk 1]
```diff
@@ -1,251 +1,264 @@
...
+    },
+    [checkStatus]
+  );
+
+  /* add task button disable */
+  const isAddButtonDisabled = (employee: IEmployee) => {
+    return (
+      employee.task.trim() === "" ||
+      employee.project === "" ||
+      (employee.task.trim() === "" && employee.project === "")
```
Comment (daxesh-parmar-scaletech) [inline src/features/checkIn/component/checkIn.tsx]: No need for third condition.

File: src/features/project/style/project.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.team {
+  .toolbar-navigation {
+    top: 84px !important;
+    right: 100px !important;
+    border-radius: 4px;
+
+    button:first-child {
+      border-radius: 4px 0px 0px 4px;
```
Comment (daxesh-parmar-scaletech) [inline src/features/project/style/project.scss]: use border-radius: 4px 0px;

File: src/features/request/components/changeTimesheetForm.tsx [hunk 3]
```diff
@@ -1,160 +1,159 @@
...
-        let formattedTime = input
-            .replace(/\D/g, '') // Remove non-numeric characters
-            .slice(0, 4) // Limit to a maximum of 4 characters (hhmm)
-            .replace(/(\d{2})(\d{0,2})/, '$1:$2'); // Add a colon between the first 2 digits
+  const handleTimeChange = (e) => {
+    const input = e.target.value;
+    let formattedTime = input
+      .replace(/\D/g, "")
+      .slice(0, 4)
+      .replace(/(\d{2})(\d{0,2})/, "$1:$2");
```
Comment (daxesh-parmar-scaletech) [inline src/features/request/components/changeTimesheetForm.tsx]: please add comments here

**Review by daxesh-parmar-scaletech — CHANGES_REQUESTED** [review]

File: src/features/checkIn/component/checkIn.tsx (hunk 1 above)
Comment (khushbuSatasiya) [reply src/features/checkIn/component/checkIn.tsx]: done

File: src/features/request/components/changeTimesheetForm.tsx (hunk 3 above)
Comment (khushbuSatasiya) [reply src/features/request/components/changeTimesheetForm.tsx]: done

File: src/features/checkOut/component/checkOut.tsx [hunk 4]
```diff
@@ -1,268 +1,239 @@
...
+				}
+
+				const payload = {
+					outTime: values.time,
+					date: checkOutDate,
+					tasks: tasks.filter((value) => value.projectHours && value.projectId && value.taskName)
+				};
+				setIsLoading(true);
+				try {
+					await httpService.post(API_CONFIG.path.checkOut, payload).then((res: any) => {
```
Comment (daxesh-parmar-scaletech) [inline src/features/checkOut/component/checkOut.tsx]: Remove response if you are not using it.

**Review by daxesh-parmar-scaletech — CHANGES_REQUESTED** [review]

File: src/features/checkOut/component/checkOut.tsx (hunk 4 above)
Comment (khushbuSatasiya) [reply src/features/checkOut/component/checkOut.tsx]: done

**Pull request merged**